### PR TITLE
Add Walnut Creek File

### PR DESCRIPTION
**Pull request by Sreeparvathy**
This is the beginning of Walnut Creek File 

Pleas provide feedback on:

- if you've been to Walnut Creek, what did you like? what did you not like?
- if you've never been to Walnut Creek, what interests you about coming here?
- @smithaharisankar checkout my branch 😉

### CODE REVIEW

File: Sreeparvathy-WalnutCreek.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# WalnutCreek, CA
+
+### Restaurants
+- Restaurant1
+- Restaurant2
```
Comment (bjswick33) [inline Sreeparvathy-WalnutCreek.md]: @Sreeparvathy Is this the one on Street1 next to building21?  lol

Comment (Sreeparvathy) [inline Sreeparvathy-WalnutCreek.md]: @bjswick33 yes opposite wicked street 😉

**Comment by readiu** [comment]
Ready to :ship:

**Comment by smithaharisankar** [comment]
Ready to 🚢

**Pull request merged**